cmd/thanos/store.go

```diff
@@ -43,7 +43,7 @@ func registerStore(app *extkingpin.App) {
 	cmd := app.Command(component.Store.String(), "Store node giving access to blocks in a bucket provider. Now supported GCS, S3, Azure, Swift, Tencent COS and Aliyun OSS.")
 
 	httpBindAddr, httpGracePeriod := extkingpin.RegisterHTTPFlags(cmd)
-	grpcBindAddr, grpcGracePeriod, grpcCert, grpcKey, grpcClientCA := extkingpin.RegisterGRPCFlags(cmd)
+	grpcBindAddr, grpcGracePeriod, grpcCert, grpcKey, grpcClientCA, tokenRate := extkingpin.RegisterGRPCFlags(cmd)
```

Comment:
@bwplotka  I experimented a bit with the flags, I think that I will need to create 2 variables and pass all around the sandwiches functions we have on the different codebase. I will need to work on this more
Unless I'm missing things..